packages/@ember/-internals/glimmer/lib/templates/outlet-helper-component.gts

```diff
@@ -0,0 +1,58 @@
+import { Component } from '@lifeart/gxt';
```

Comment:
why not `@glimmer/component` (not the real glimmer component, but re-defined in this PR, using glimmer-next apis?)

Reply:
need to do some copy-paste from non-released branch - https://github.com/lifeart/glimmer-next/pull/25